ddtrace/writer.py

```diff
@@ -151,6 +156,11 @@ def _target(self):
                 # no traces and the queue is closed. our work is done
                 return
 
+            if hasattr(result_traces, 'read'):
+                result_traces_body = result_traces.read()
+                if hasattr(self._priority_sampler, 'set_sample_rates_from_json'):
```

Comment:
https://github.com/DataDog/dd-trace-py/commit/cc47f23132845fc68c1038e3c3ed4eda0ed6dfc9 makes this obsolete. AFAIK it was here initially because I had plans to technically support users overriding the sampler with there own impl. As now we only support `priority_sampling=True` and not `priority_sampler=MyOwnSampler` we're fine.